addon/styles/_frost-chart-svg-plot-grid.scss

```diff
@@ -1,6 +1,9 @@
-.frost-chart-svg-plot-grid {
-  line {
-    stroke: $frost-color-grey-7;
-    stroke-width: 1px;
+.frost-chart-svg-plot {
+  &-x-grid,
+  &-y-grid {
```

Comment:
Minor, but something that Quincy pointed out before and I agree with - it's hard to search for the class name when using &, so I would prefer to reserve & for peer classes - e.g.

```
.my-class {
  &.modifier {
```

In this case I would do the more explicit

```
.frost-chart-svg-plot-x-grid,
.frost-chart-svg-plot-y-grid
```